.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -38,35 +38,143 @@ jobs:
-          MO_GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          MO_GH_PKG_REPO: 'datastax/cass-operator/operator'
-        run: |
+          fi
+
           export PATH=$GOROOT/bin:$GOPATH/bin:$PATH
-          export MO_TAGS=$(cat ./build/tagsToPush.txt)
-          mage operator:deployToGHPackages
+          GITHUB_REPO_URL="https://github.com/${{ github.repository }}"
+
+          VERSION="$(mage operator:printVersion)"
+          FULL_VERSION="$(mage operator:printFullVersion)"
+          VERSION_STAMP="${{ github.repository }}:${FULL_VERSION}"
+
+          ECR_REPOSITORY="${ECR_REPO}/datastax/cass-operator"
+          GH_REPOSITORY="docker.pkg.github.com/${{ github.repository_owner }}/cass-operator/operator"
+
+          ECR_TAGS=()
+          ECR_UBI_TAGS=()
+          GH_TAGS=()
+          GH_UBI_TAGS=()
+          GH_ARM64_TAGS=()
+
+          for t in "${FULL_VERSION}" "${GITHUB_SHA}" "latest"; do
+            ECR_TAGS+=(--tag "${ECR_REPOSITORY}:${t}")
+            ECR_UBI_TAGS+=(--tag "${ECR_REPOSITORY}:${t}-ubi")
+
+            GH_TAGS+=(--tag "${GH_REPOSITORY}:${t}")
+            GH_UBI_TAGS+=(--tag "${GH_REPOSITORY}:${t}-ubi")
+            GH_ARM64_TAGS+=(--tag "${GH_REPOSITORY}:${t}-arm64")
+          done
+
+          LABELS=(
+            --label "org.label-schema.schema-version=1.0"
+            --label "org.label-schema.vcs-ref=$GITHUB_SHA"
+            --label "org.label-schema.vcs-url=$GITHUB_REPO_URL"
+            --label "org.label-schema.version=$VERSION"
+          )
+
+          COMMON_ARGS=(
+            "${LABELS[@]}"
+            --file operator/docker/base/Dockerfile
+            --cache-from "type=local,src=/tmp/.buildx-cache"
+            --cache-to "type=local,dest=/tmp/.buildx-cache"
+          )
+
+          STANDARD_ARGS=(
+            "${COMMON_ARGS[@]}"
+            --label "release=${VERSION_STAMP}"
+            --build-arg "VERSION_STAMP=${VERSION_STAMP}"
+            --target cass-operator
+          )
+
+          UBI_ARGS=(
+            "${COMMON_ARGS[@]}"
+            --label "release=${VERSION_STAMP}-ubi"
+            --build-arg "VERSION_STAMP=${VERSION_STAMP}-ubi"
+            --build-arg "BASE_OS=registry.access.redhat.com/ubi7/ubi-minimal:7.8"
+            --target cass-operator-ubi
+          )
+
+          # Build and push standard images
+
+          docker buildx build \
+            --push \
+            "${STANDARD_ARGS[@]}" \
+            "${ECR_TAGS[@]}" \
+            --platform linux/amd64,linux/arm64 \
+            .
+
+
+          # Build and push UBI images
+
+          docker buildx build \
+            --push \
+            "${UBI_ARGS[@]}" \
+            "${ECR_UBI_TAGS[@]}" \
+            --platform linux/amd64 \
+            .
+
+
+          # Workaround for GH packages
+
+          docker buildx build \
+            --load \
+            "${STANDARD_ARGS[@]}" \
+            "${GH_TAGS[@]}" \
+            --platform linux/amd64 \
+            .
```

Comment:
GH packages don't support multiarch images, or a lot of other things. (This _should_ be fixed in GitHub's new docker registry beta, but I haven't tested it. It does fix containerd though.) This means we have to push images with different tags for different architectures for GH packages. Also, buildx has trouble pushing directly to GH packages... so we have to load the images into docker local images and push them using `docker push`.